src/containers/AssignmentTexterContact.jsx

```diff
@@ -440,18 +440,29 @@ export class AssignmentTexterContact extends React.Component {
       return // stops from multi-send
     }
     this.setState({ disabled: true })
-    try {
-      await this.props.mutations.sendMessage(message, contact.id)
-      const optOut = {
-        cell: contact.cell,
-        assignmentId: assignment.id
-      }
 
+    if(optOutMessageText.length > 0){
+      try {
+        await this.props.mutations.sendMessage(message, contact.id)
```

Comment:
I think this would be more readable if we just remove the top condition and then wrap the `> 0` condition around the sendMessage.